docs/sys-admin/migration-and-upgrade.md

```diff
@@ -0,0 +1,247 @@
+---
+title: Migration and Upgrade
+toc_max_heading_level: 5
+---
+
+# Migration and Upgrade
+
+This guide provides detailed instructions for two scenarios:
+1. Migrating from Greenplum Database 6.x/7.x to Cloudberry Database
+2. Upgrading from Cloudberry Database 1.x to Cloudberry Database 2.x
```

Comment:
```suggestion
2. Upgrading from Apache Cloudberry 1.x (non-Apache release before joining the Apache Incubator) to Apache Cloudberry 2.x
```

Also, we need to replace the `Cloudberry Database` with `Apache Cloudberry` in the docs.